### DISCUSSION TITLE
anti kick nameless

### DISCUSSION THREAD

**Discussion by NejatJafarli** [winstxnhdw/lc-hax#471] [Q&A] [answered]
there have any way to only use anti kick but not became nameless 

i already removed other mods in loader.cs 2 mods enabled MinimalGUI and Antikick but still I am became nameless

**Comment by winstxnhdw** [comment]
Nameless to others or to yourself?

**Reply by NejatJafarli** [reply]
Other persons see my name nameless

**Reply by Totoqoe** [reply, marked as answer]
I feel like that's part of the appeal as it hides your actual account, but if you really wanted your name to appear for others, join the lobby with antikick off first and then rejoin with it on.

**Reply by winstxnhdw** [reply]
What Toto said. This is intentional.
